lib/netstd/Thrift/Protocols/Utilities/TBase64Helper.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the Apache Software Foundation(ASF) under one
+// or more contributor license agreements.See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+
+namespace Thrift.Protocols.Utilities
+{
+    // ReSharper disable once InconsistentNaming
+    internal static class TBase64Helper
```

Comment:
Good point. The current implementation allocates minimal space, which might be useful when dealing with larger binary values. Nevertheless that should be revisited later, since there may be indeed some potential for optimization. I'd rather not change it now, since that code is known to work and I don't want to generate more lose ends that need to be fixed for now.